### PR TITLE
Tess dev hub setup

### PR DESCRIPTION
**Pull request by jaytmiller**
Fixes secrets scripts broken by move of secrets directory.
Moves e-mails from config/common.yaml to the secrets template.
Sets TESS memory allocations to 4G guaranteed, 10G limit basically assuming t3.xlarge.
Adds new EFS configuration to TESS common.yaml,  roman still needs it.
Adds ssh session timeout and Terraform logging env variables to setup-env.template.
Update TESS EFS configuration in common.yaml to avoid creation of EBS,  node affinity conflicts
Switched image tag from <mission>-latest back to latest in TESS, Roman, setup-env.template.

### CODE REVIEW

File: deployments/roman/config/common.yaml [hunk 1]
```diff
@@ -35,9 +35,3 @@ jupyterhub:
     memory:
       guarantee: 8G
       limit: 15G
-  auth:
```
Comment (super-cob) [inline deployments/roman/config/common.yaml:38]: can we put this in secrets instead?

Comment (jaytmiller) [reply deployments/roman/config/common.yaml:38]: Yep,  that's what I did.  I removed it from common.yaml and added it to the secrets template.  It worked fine.  I removed it from common.yaml for both tess and roman.

Comment (jaytmiller) [reply deployments/roman/config/common.yaml:38]: (It worked fine with real e-mails...)

**Comment by jaytmiller** [comment]
Resolved all comments with Jake, merging.

**Pull request merged**